server_admin/topics/users/viewing.adoc

```diff
@@ -1,19 +1,24 @@
+[searching-users]
+= Searching for users
 
-=== Searching For Users
+You can search for a user in the database, by using the full name, last name, or email address of the user.
 
-If you need to manage a specific user, click on `Users` in the left menu bar.
+.Prerequisites
+* You are logged in to the correct realm and are not in the master realm.
 
-.Users
-image:{project_images}/users.png[]
+.Procedure
+. Click on *Users* in the left menu bar. This will direct you to the user list page.  
+. In the search box, type the full name, last name, or email address of the user you want to search for in the user database. The query will bring up all users that match your criteria.  
```

Comment:
No need for ".. in the user database" unless it is relevant. 
Also
"The search returns ...."